index.spec.js

```diff
@@ -0,0 +1,203 @@
+const supertest = require('supertest');
+const app = require('./index');
+const http = require('http');
+const { User, Hospital } = require('./db/models');
+
+jest.mock('./controllers/utils');
+const { findCoordinates } = require('./controllers/utils');
+
+let server;
+let request;
+
+beforeEach(done => {
+  server = http.createServer(app);
+  server.listen(done);
+  request = supertest(server);
+});
+
+afterEach(done => {
+  server.close(done);
+});
+
+describe('/users', () => {
+  const endpoint = '/users';
+  let user = {};
+  let hospital = {};
+
+  beforeAll(async () => {
+    hospital = await Hospital.create({
+      name: 'Plural Engine Hospital',
+      address: 'C/ Pau Alsina 123',
```

Comment:
I’d rather very much we didn’t push real information. We need to delete this commit history

Reply:
```suggestion
      address: 'Lolipop street',
```